readme.txt

```diff
@@ -75,7 +96,7 @@ All inputs are [sanitized and escaped](https://developer.wordpress.org/themes/th
 * **Compatible with all WooCommerce themes.**
 
 
-[See All PRO Features](https://themeisle.com/plugins/ppom-pro/upgrade/?utm_source=themeisle&utm_medium=product&utm_campaign) - [Demos](https://demo-ppom-lite.vertisite.cloud/)
+[See All PRO Features](https://themeisle.com/plugins/ppom-pro/upgrade/?utm_source=themeisle&utm_medium=product&utm_campaign) - [Demos](https://demo-ppom-lite.vertisite.cloud/) - [Ask a Pre-sale Question](https://themeisle.com/contact/?utm_source=wordpress.org&utm_medium=product&utm_campaign=about-us)
```

Comment:
Regarding UTMs you can use the same values/pattern as `tsk_utmify` function, I saw that the free vs pro also has a wrong utm too. 

Also, why is the pre-sales link something that we want to add? I would revert that for now.